paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewState.kt

```diff
@@ -28,16 +25,8 @@ internal sealed class CustomerSheetViewState(
 ) {
     abstract fun topBarState(onEditIconPressed: () -> Unit): PaymentSheetTopBarState
 
-    fun shouldDisplayDismissConfirmationModal(
-        isFinancialConnectionsAvailable: IsFinancialConnectionsAvailable,
-    ): Boolean {
-        return this is AddPaymentMethod &&
-            paymentMethodCode == PaymentMethod.Type.USBankAccount.code &&
-            isFinancialConnectionsAvailable() &&
-            bankAccountResult is CollectBankAccountResultInternal.Completed &&
-            bankAccountResult.response.usBankAccountData
-                ?.financialConnectionsSession
-                ?.paymentAccount is FinancialConnectionsAccount
```

Comment:
This logic isn’t present on iOS, and it feels weird anyway to treat verified and non-verified bank accounts differently.